ci/tempest-multinode/control_plane_hook.yaml

```diff
@@ -24,3 +24,8 @@
               - op: replace
                 path: /spec/heat/enabled
                 value: true
+              - op: add
+                path: /spec/heat/template/customServiceConfig
+                value: |
+                  [clients_heat]
+                  endpoint_type: public
```

Comment:
it fixes the SSL issues

Reply:
actually the SSL thing was a real issue, and it is addressed here: https://github.com/openstack-k8s-operators/heat-operator/pull/433